executor/executor_test.go

```diff
@@ -6309,3 +6309,57 @@ func TestIssue39211(t *testing.T) {
 	tk.MustExec("set @@tidb_enable_null_aware_anti_join=true;")
 	tk.MustQuery("select * from t where (a,b) not in (select a, b from s);").Check(testkit.Rows())
 }
+
+func TestIssue39618(t *testing.T) {
```

Comment:
I suggest that this test is put into ```executor/issuetest```. Too many test cases were put into the executor folder. it will slow the performance of bazel test, especially when it meets a flaky test.

Reply:
BTW, executor/issuetest is in existed at release-6.5/